src/stack_aggregator.py

```diff
@@ -578,10 +574,19 @@ def execute(aggregated=None, persist=True):
             '_release': 'None:None:None'
         }
         if persist:
-            return persist_data_in_db(external_request_id=external_request_id,
-                                      task_result=stack_data, worker='stack_aggregator_v2',
-                                      started_at=started_at, ended_at=ended_at)
+            persiststatus = persist_data_in_db(external_request_id=external_request_id,
+                                               task_result=stack_data, worker='stack_aggregator_v2',
+                                               started_at=started_at, ended_at=ended_at)
+            # Ingestion of Unknown dependencies
+            for dep in unknown_dep_list:
```

Comment:
The same code is repeated in the if and else clause. The code can be shifted outside the if/else, before the return statement.